src/components/Default/index.js

```diff
@@ -22,14 +22,17 @@ const parseNav = (items, lang, slug) => (
       .map(item =>
         <Fragment key={item.slug}>
           <li>
-            <Link
-              className={item.active ? 'text-theme-color strong' : ''}
-              to={addLangToUrl(item.slug, lang)}
-            >
-              {item.title}
-            </Link>
+            {item.nav
+              ? <h3>{item.title}</h3>
```

Comment:
Might be opinionated, but are you sure you don't want to link to the subpage for the title item? Since the page will exist and could contain content

Reply:
I'm sure. The only plans have for this currently are for the committees page, dividing into organs, and we don't have anything to say about each organ. We just want the committee pages to be divided under headings.